src/controllers/executionPlanWebviewController.ts

```diff
@@ -12,6 +12,7 @@ import { homedir } from "os";
 import { exists } from "../utils/utils";
 import UntitledSqlDocumentService from '../controllers/untitledSqlDocumentService';
 import * as path from 'path';
+import { ApiStatus } from "../sharedInterfaces/connectionDialog";

```

Comment:
should execution plan take a dependency on connection dialog?  i.e. should we move ApiStatus to a common shared location?

Reply:
Will move to a new file since it is a common interface